benchmark/Benchmark/HtomlMegaparsec.hs

```diff
@@ -3,17 +3,24 @@
 {-# LANGUAGE PackageImports #-}
 
 module Benchmark.HtomlMegaparsec
-       ( convert
-       , decode
+       ( decode
        , parse
+       , convert
        ) where
 
 import Data.Aeson.Types (ToJSON, parseEither, parseJSON, toJSON)
+import Data.Semigroup ((<>))
 import Data.Text (Text)
+
 import "htoml-megaparsec" Text.Toml (Node (..), Table, TomlError, parseTomlDoc)
 
 import Benchmark.Type (HaskellType)
 
+-- | Decode toml file to Haskell type.
+decode :: Text -> Either String HaskellType
+decode txt = case parseTomlDoc "log" txt of
```

Comment:
I see these functions in every module, can we make them polymorphic?

Reply:
Hmm, I haven't thought about this :thinking: It's not that simple because the use `convert` and `parse` specific to every library. But maybe it's possible to somehow remove code duplication...